docs/api/api/api.txt

```diff
@@ -712,6 +712,22 @@ Parameters:
 XmlResult: status
 
 
+POST /source/<project>/<package>?cmd=fork&scmsync=url
```

Comment:
I'm not sure I'd like to introduce another term for this. Can't we just make this an optional parameter for branch?